README.md

```diff
@@ -794,6 +794,24 @@
 
     ```
 
+  - Remove all trailing whitespace
+
+    ```javascript
+    // bad
+    function() {∙∙
+      var name;
+    ∙∙
+      // ... stuff ...
+    }
+
+    // good
+    function() {
+      var name;
+
+      // ... stuff ...
+    }
+    ```
+
```

Comment:
The tools exist although they're not as comprehensive:

https://github.com/jshint/fixmyjs
https://github.com/millermedeiros/esformatter